app/view-plants/page.tsx

```diff
@@ -0,0 +1,110 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { UUID } from 'crypto';
+import supabase from '@/api/supabase/createClient';
+import { get_plant_by_id } from '@/api/supabase/queries/plant_by_id';
+import { getPlantSeasonality } from '@/api/supabase/queries/plantSeasonality';
+import PlantCard from '@/components/PlantCard/PlantCard';
+import { Plant } from '@/types/schema';
+
+export default function Home() {
+  const [selectedOption, setSelectedOption] = useState<'myPlants' | 'all'>(
```

Comment:
nit: we could also consider renaming/refactoring this state to 
`const [setSeeMyPlants, setSeeMyPlants] = useState<boolean>(true)`
However, the current implementation is also nice for clarity. If we stick with the current state options, we can still rename the state to `viewingOption`/`setViewingOption` instead of selectedOption
(Eventually, the default screen will depend on `has_plot` in the ProfileContext)